modules/S22_php_check.sh

```diff
@@ -120,31 +143,8 @@ s22_vuln_check() {
       COMMON_FILES_FOUND=""
       CFF="NA"
     fi
-    print_output "[+] Found ""$ORANGE""$VULNS"" vulnerabilities""$GREEN"" in php file"": ""$ORANGE""$(print_path "$LINE")""$GREEN""$COMMON_FILES_FOUND""$NC" "" "$PHP_LOG"
-    write_csv_log "$(print_path "$LINE")" "$VULNS" "$CFF"
-    echo "$VULNS" >> "$TMP_DIR"/S22_VULNS.tmp
-  fi
-}
-
-# lets leave this here. Probably it is of interest for dev teams
-# for this you need to call it (see line 32/35)
-s22_script_check() {
```

Comment:
❔ intended delete?

Reply:
yes ... nobody has requested to activate it again for a while. So, lets remove the old code now